src/account/AccountItem.js

```diff
@@ -42,29 +40,75 @@ export default class AccountItem extends React.PureComponent {
   };
 
   handleSelect = () =>
-    this.props.onSelect(this.props.index);
+  this.props.onSelect(this.props.index);
 
   handleRemove = () =>
-    this.props.onRemove(this.props.index);
+  this.props.onRemove(this.props.index);
 
   render() {
-    const { email, realm } = this.props;
+    const { email, realm, index } = this.props;
 
     return (
       <Touchable style={styles.wrapper} onPress={this.handleSelect}>
-        <View style={styles.accountItem}>
+        <View style={[styles.accountItem, this.getDynamicStyles(index)]}>
```

Comment:
This might be simplified if we only add a special style for the 'selected' account instead of calling functions. Something like `[styles.accountItem, index === 0? styles.selected : {}]`